src/pages/Inventory.page.js

```diff
@@ -9,7 +9,34 @@ export class InventoryPage extends BaseSwagLabPage {
 
     addItemToCartButton = this.page.locator('[id^="add-to-cart"]');
 
+    dropdown = this.page.locator('.product_sort_container');
```

Comment:
Lets use https://playwright.dev/docs/api/class-framelocator#frame-locator-get-by-test-id 
This is more preferable way to find elements in Playwright
The same for other locators